### ISSUE TITLE
mapStateToPropsWithOwnProps not called properly on OverReactRedux connected component

### ISSUE THREAD

**Issue by dave-doty** [Workiva/over_react#425]
> * Issue Type: BUG
> * `over_react` Version(s): `3.1.4`

It appears that `mapStateToPropsWithOwnProps` is not called (or is called improperly) when passed into the `connect` function to create a connected component with OverReact Redux.

To reproduce in the [OverReact Redux simple example](https://github.com/Workiva/over_react/tree/master/web/over_react_redux/examples/simple), follow these steps:

1) Add the following to components/counter.dart:

```dart
UiFactory<CounterProps> ConnectedMultiplierCounter = connect<CounterState, CounterProps>(
    mapStateToPropsWithOwnProps: (state, props) =>
        (Counter()..currentCount = state.smallCount * props.multiplier))(Counter);
```

at the top-level, and add a new prop `int multiplier` in `_$CounterProps`.

2) Add the following lines to (e.g., in between lines [18](https://github.com/Workiva/over_react/blob/master/web/over_react_redux/examples/simple/index.dart#L18) and [19](https://github.com/Workiva/over_react/blob/master/web/over_react_redux/examples/simple/index.dart#L19)) in index.dart:

```dart
Dom.h2()('ConnectedMultiplierCounter'),
(ConnectedMultiplierCounter()..multiplier = 7)(),
```

3) Add the following lines to the `render()` method of components/counter.dart:

```dart
print('rendering CounterComponent');
print('  multiplier = ${props.multiplier}');
print('  currentCount = ${props.currentCount}');
```

The following should be printed (in some order) when `ConnectedMultiplierCounter` is rendered:

```
rendering CounterComponent
  multiplier = 7
  currentCount = null
```

whereas the expected behavior is to print

```
rendering CounterComponent
  multiplier = 7
  currentCount = 7
```

---

> __FYI:__ @greglittlefield-wf @aaronlademann-wf @kealjones-wk @evanweible-wf @maxwellpeterson-wf

**Comment by greglittlefield-wf** [comment]
Hi @dave-doty, sorry about that!! We actually fixed that in https://github.com/Workiva/over_react/pull/423, but haven't released it yet 😬 . I'll get a release going

**Comment by greglittlefield-wf** [comment]
Fix released in over_react 3.1.5
